plugin/gudid-transformation-maven-plugin/src/main/java/dev/ikm/maven/GudidUtility.java

```diff
@@ -220,4 +221,9 @@ public EntityProxy.Concept getModuleConcept() {
         return EntityProxy.Concept.make(PublicIds.of("8449f4f5-1a96-478a-864a-232f3afa0ee6"));
     }
 
+    public EntityProxy.Concept lookupDeviceIdIssuingAgencyConcept(String deviceIdIssuingAgencyCode) {
+        String issuingAgencyName = getDeviceIdIssuingAgencyName(deviceIdIssuingAgencyCode);
+        return EntityProxy.Concept.make(issuingAgencyName, UuidT5Generator.get(namespace, issuingAgencyName));
```

Comment:
See above comment about the need for an underlying concept in the database that matches the issuingAgencyName concept proxy